packages/api/src/brevo-api/brevo-api-campaigns.service.ts

```diff
@@ -12,9 +16,26 @@ import { BrevoApiCampaignStatistics } from "./dto/brevo-api-campaign-statistics"
 export class BrevoApiCampaignsService {
     private readonly campaignsApi: SibApiV3Sdk.EmailCampaignsApi;
 
-    constructor(@Inject(BREVO_MODULE_CONFIG) private readonly config: BrevoModuleConfig) {
+    constructor(
+        @Inject(BREVO_MODULE_CONFIG) private readonly config: BrevoModuleConfig,
+        @InjectRepository("BrevoConfig") private readonly brevoConfigRepository: EntityRepository<BrevoConfigInterface>,
```

Comment:
I think `"BrevoConfig"` should be a constant defined in `brevo-config-entity.factory.ts`